### PR TITLE
Add footer and fix some styles

### PR DESCRIPTION
**Pull request by JuannFerrari**
- This PR adds a new footer to the playground and fix some issues related to the sidebar + new styles

**BEFORE:**
Without footer:
Screen Shot 2021-07-15 at 17 09 49
Sidebar weird collapse:
Screen Shot 2021-07-15 at 17 10 21

AFTER:
New footer:
Screen Shot 2021-07-15 at 17 10 46
Sidebar fixed:
Screen Shot 2021-07-15 at 17 11 25

### CODE REVIEW

File: app/views/boards/_form.html.erb [hunk 1]
```diff
@@ -1,10 +1,6 @@
-<%= form_with(model: board, class: 'mt-5 flex flex-col') do |form| %>
-  <div>
-    <%= form.label :name %>
-    <%= form.text_field :name, class: 'shadow' %>
-  </div>
-
-  <%= form.button(:submit, class: 'button mt-5 w-1/5') do %>
+<%= form_with(model: board, class: 'mt-5 flex flex-col w-2/4') do |form| %>
+  <%= render(FieldComponent.new(:name, form: form, placeholder: 'Name', required: true)) %>
+  <%= form.button(:submit, class: 'button mt-5') do %>
```
Comment (JulianPasquale) [inline app/views/boards/_form.html.erb]: This is just a question, I'm not sure if this actually make sense but can we use `ButtonComponent` here?

**Review by JulianPasquale — APPROVED** [review]
❤️ 🚀

**Review by jdrosales17 — APPROVED** [review]
🙌

File: app/views/boards/_form.html.erb (hunk 1 above)
Comment (JuannFerrari) [reply app/views/boards/_form.html.erb]: Totally, nice catch!

**Pull request merged**